### ISSUE TITLE
When integrating via cocoa pods, G8 prefixed headers are not present in framework

### ISSUE THREAD

**Issue by emablekos** [gali8/Tesseract-OCR-iOS#111]
On the left is the cocoapods framework, on the right is the framework from the products directory in the repo. Sample code depends on g8 classes.

![2014-12-22 at 10 21 pm](https://cloud.githubusercontent.com/assets/299982/5534288/76320404-8a2a-11e4-8f13-ed4642d8a03a.png)

**Comment by joeldrotleff** [comment]
Just wondering if you can suggest a method of preventing cocoa pods from re-downloading the repository every time I do a pod install. Seems like because it's set to download from github as you suggest, the normal caching is disabled.

**Comment by kevincon** [comment]
@joeldrotleff you can clone the GitHub repository somewhere local on your computer, and then in your podfile use:

```
pod 'TesseractOCRiOS', :path => "/path/to/Tesseract-OCR-iOS"
```

**Comment by joeldrotleff** [comment]
Ah that works great. Thanks!

**Comment by kevincon** [comment]
Fixed by #123.
